core/src/mast/debuginfo/decorator_storage.rs

```diff
@@ -57,18 +60,18 @@ use crate::mast::{DecoratedOpLink, DecoratorId, MastNodeId};
 pub struct OpToDecoratorIds {
     /// All the decorator IDs per operation per node, in a CSR relationship with
     /// node_indptr_for_op_idx and op_indptr_for_dec_ids
-    decorator_ids: Vec<DecoratorId>,
+    pub(crate) decorator_ids: Vec<DecoratorId>,
```

Comment:
This and other fields only need to be `pub(super)` right?

Also, why not just implement `Serializable` and `Deserializable` for `OpToDecoratorIds` rather than exposing these fields?